sui/typescript/dapp_kit_example/src/client/App.tsx

```diff
@@ -0,0 +1,175 @@
+import "./App.css";
+import { useState } from "react";
+import { 
+  ConnectButton, 
+  useCurrentAccount,
+  useSignTransaction,
+  useSuiClient
+} from "@mysten/dapp-kit";
+import { Box, Flex, Heading } from "@radix-ui/themes";
+import axios from 'axios';
+import { SuiTransactionBlockResponse } from "@mysten/sui/client";
+import { useShinamiClient }  from "./hooks/useShinamiClient.js";
+
+function App() {
+  const currentAccount = useCurrentAccount();
+  const { mutateAsync: signTransaction } = useSignTransaction();
+  const client = useSuiClient();
```

Comment:
This doesn't appear to be used

Reply:
If I take out the `useShinamiClient()` I'll go back to using this.